eng/Packages.Data.props

```diff
@@ -281,6 +277,8 @@
     <PackageReference Update="NUnit" Version="3.13.2" />
     <PackageReference Update="NUnit3TestAdapter" Version="4.4.2" />
     <PackageReference Update="OpenTelemetry" Version="1.6.0" />
+    <PackageReference Update="OpenTelemetry.Instrumentation.AspNetCore" Version="1.5.1-beta.1" />
```

Comment:
Should we remove these for now?

Reply:
I moved this to the test package location. Client library cannot take reference on it. This is used by our integration tests project.